src/app/store.ts

```diff
@@ -1,6 +1,7 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { currentTodoSlice, filterSlice, todosSlice } from '../features';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(currentTodoSlice, filterSlice, todosSlice);
```

Comment:
The function `combineSlices` is likely incorrect. The correct function to combine reducers in Redux Toolkit is `combineReducers`. Please verify the function name and update it if necessary.